treeherder/webapp/api/performance_data.py

```diff
@@ -711,7 +711,7 @@ def __filter_out_retriggers(serialized_data: List[dict]) -> List[dict]:
                     seen_push_id = datum['push_id']
 
             start, end = delectable_idxes[0], delectable_idxes[-1]
```

Comment:
Shouldn't we put an `if` condition here? We surely have signatures without any retriggers. This should cause an `IndexError` when unpacking the `delectable_idxes` variable.